subsys/net/ip/tcp.c

```diff
@@ -2193,6 +2193,10 @@ int net_tcp_queue_data(struct net_context *context, struct net_pkt *pkt)
 	k_mutex_lock(&conn->lock, K_FOREVER);
```

Comment:
Minor typo in commit `net: tcp: Simple zero window probe implementation`
s/Proble/Probe/